Assets/Editor/BuildTiltBrush.cs

```diff
@@ -1218,6 +1218,7 @@ public static void DoBuild(TiltBuildOptions tiltOptions)
         string stamp = tiltOptions.Stamp;
         SdkMode vrSdk = tiltOptions.VrSdk;
         BuildOptions options = tiltOptions.UnityOptions;
+        // Add your new scenes in this List for your app for Oculus Quest 
```

Comment:
You should delete the 'for Oculus Quest' here as the same applies to PC builds as well.

Also, [optional] it might be better to rephrase this as something like:

```
// During the build process the Scene List in the Build Settings is ignored.
// Only the following scenes are included in the build.
```

... as that describes _what the code does_, rather than _what the reader should do_.

Reply:
I really appreciate your reaction !!
I changed the notes with your idea.
I'm so excited with the chance to contribute this great project !!